spiderable_server.js

```diff
@@ -4,6 +4,10 @@ var querystring = Npm.require('querystring');
 var urlParser = Npm.require('url');
 var crypto = Npm.require('crypto');
 
+var bound = Meteor.bindEnvironment(function(callback) {
```

Comment:
bound is really ambiguous.  I'd rename it to something that explicitly identifies what it is doing, such as:
runAsync or execAsync or wrapAsync.
Also, seems like we should use Meteor.wrapAsync instead.
http://docs.meteor.com/#/full/meteor_wrapasync